packages/components/src/InputTime/InputTime.tsx

```diff
@@ -45,22 +19,26 @@ export function InputTime({
     onChange && onChange(htmlTimeToCivilTime(newValue));
   };
 
-  const fieldProps: FormFieldProps = {
-    onChange: handleChange,
-    defaultValue:
-      defaultValue != undefined ? civilTimeToHTMLTime(defaultValue) : undefined,
-    value: value != undefined ? civilTimeToHTMLTime(value) : undefined,
-    ...params,
-  };
-
-  return <FormField type="time" {...fieldProps} />;
-}
-
-function civilTimeToHTMLTime(civilTime: CivilTime) {
-  const timeString = civilTime.toString();
-  return timeString.substring(0, timeString.indexOf("."));
-}
-
-function htmlTimeToCivilTime(timeString: string) {
-  return CivilTime.fromString(timeString + ":00.000000000");
+  if (supportsTime) {
+    const fieldProps: FormFieldProps = {
+      onChange: handleChange,
+      defaultValue:
```

Comment:
might I suggest a conditional spread operators

```suggestion
      ...(defaultValue && { defaultValue: civilTimeToHTMLTime(defaultValue) }),
```